test/docs.js

```diff
@@ -189,7 +191,7 @@ describe('documentation', function() {
 
           // in:  fs.anyMethod('--any-file-path--
           // out: fs.anymethod('--a-temporary-file--
-          .replace(/(fs\.[^(]+\(['"])([^'"]*)/g, '$1' + temporaryFile.name);
```

Comment:
> Instead of rewriting the code, what about mocking fs and intercepting the filename?

I like that idea as well, but the downside is we have to recreate all of the `fs` methods -- we'd have to figure out what is our using of `fs` vs. some other internal API that might actually need it to work.